src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf

```diff
@@ -334,6 +334,38 @@
                 <source>This value should be valid JSON.</source>
                 <target>Ova vrednost bi trebalo da bude validan JSON.</target>
             </trans-unit>
+            <trans-unit id="87">
+                <source>This collection should contain only unique elements.</source>
+                <target>Ova kolekcija bi trebala da sadrži samo jedinstvene elemente.</target>
+            </trans-unit>
+            <trans-unit id="88">
+                <source>This value should be positive.</source>
+                <target>Ova vrednost bi trebala biti pozitivna.</target>
+            </trans-unit>
+            <trans-unit id="89">
+                <source>This value should be either positive or zero.</source>
+                <target>Ova vrednost bi trebala biti pozitivna ili nula.</target>
+            </trans-unit>
+            <trans-unit id="90">
+                <source>This value should be negative.</source>
+                <target>Ova vrednost bi trebala biti negativna.</target>
+            </trans-unit>
+            <trans-unit id="91">
+                <source>This value should be either negative or zero.</source>
+                <target>Ova vrednost bi trebala biti pozitivna ili nula.</target>
+            </trans-unit>
+            <trans-unit id="92">
+                <source>This value is not a valid timezone.</source>
+                <target>Ova vrednost nije validna vremenska zona.</target>
+            </trans-unit>
+            <trans-unit id="93">
+                <source>This password has been leaked in a data breach, it must not be used. Please use another password.</source>
+                <target>Ova lozinka je procurela prilikom prethodnih napada. Nemojte je koristiti.</target>
```

Comment:
`procurela` nekako mi ne zvuci ok a i `Please use another password` nisi preveo

Reply:
Da u vezi `procurela` pokušao sam na više načina da prevedem ali je problem prevesti ali mi je ovo imalo najviše smisla. Imaš li možda neku drugu ideju?

Reply:
Sad ću ispraviti ostatak prevoda.

Reply:
Možda ovako `Ova lozinka je kompromitovana u prethodnim napadima...`?

Reply:
to zvuci mnogo bolje

Reply:
Done :)